src/assets/translations/en/main.json

```diff
@@ -234,6 +234,7 @@
     "walletBalance": "Wallet Balance",
     "earnBalance": "Earn Balance",
     "earnBody": "Earn passive income by staking your assets or depositing them into a DeFi strategy.",
+    "earnCopy": "Click here to Earn 60%+ APR* by farming your ETH-FOX liquidity pool tokens",
```

Comment:
That's fine for now, but could we get this copy to accept an argument (hardcoded for now) so we can iterate on it in the future and add a more accurate APR? You can see the syntax for it in https://github.com/nayaabkhan/react-polyglot

Reply:
I've applied a change to accommodate for this so its no longer hardcoded.